crates/turborepo-lib/src/turbo_json/loader.rs

```diff
@@ -246,6 +285,41 @@ fn load_from_root_package_json(
     Ok(turbo_json)
 }
 
+fn root_turbo_json_from_scripts(scripts: &[String]) -> Result<TurboJson, Error> {
+    let mut turbo_json = TurboJson {
+        ..Default::default()
+    };
+    for script in scripts {
+        let task_name = TaskName::from(script.as_str()).into_root_task();
+        turbo_json.tasks.insert(
+            task_name,
+            Spanned::new(RawTaskDefinition {
```

Comment:
Having trouble getting spans for the script names as `BTreeMap` requires `impl Ord for Spanned<T>` which has issues since it contains `Range` which doesn't implement `Ord`.

I don't think this is worth the effort to plumb through. The only place this span would get used would be for [validating no package task syntax](https://github.com/vercel/turborepo/blob/main/crates/turborepo-lib/src/turbo_json/mod.rs#L671) and having that error point to a `package.json` isn't helpful when we're the ones generating `turbo.json`.